### ISSUE TITLE
Naming: change name of command from `noop` to `no-operation` or `nooperation`

### ISSUE THREAD

**Issue by pdelfino** [atlas-engineer/nyxt#1659]
The opinion below might be subjective/biased on background, since I have never used/heard/seen the "noop" word before. 
Still, I think the command  `noop` could have a better name.

1 -  "noop" or "nop" or "no-op" are all abbreviations for "no operations". Nyxt's parlance avoids abbreviations. It is probably the only command using one. Also having 3 possibilities shows the lack of consistency;

2 - abbreviations are usually written with capital letters. The Wikipedia entry for this word in Assembly domain (probably the origin) used capital letters and chose "NOP" as the main [abbreviation](https://en.wikipedia.org/wiki/NOP_(code)). The Nyxt command `noop`  is not written with capital letters and will not be since command names are lower case and fuzzy search is case sensitive.

Thus, I would suggest using `nooperation` (symmetric to `nosound`, `noimage`, `noscript`, `nowebgl`) or even `no-operation` (the clearest, imho). 

I would be happy to send a patch if you think it is valuable.

**Comment by Ambrevar** [comment]
Maybe `void` or `nothing` would be better and shorter names.  Other suggestions?

**Comment by pdelfino** [comment]
I really like the natural language expressiveness of nothing.

**Comment by pdelfino** [comment]
Something has just occured to me. I have another suggestion.

What about `unbound`?

That would be expressive in the config file:

```lisp
(define-configuration buffer
  ((override-map (let ((map (make-keymap "override-map")))
                   (define-key map
                      "C-space" 'unbound
                      "M-x" 'execute-command)))))
```

**Comment by Ambrevar** [comment]
True, `unbound` sounds good.

But note the subtlety here: with `unbound` the user might assume that
the key is unbound in this keymap and thus forwarded.  Which is not
true.  When the key is _bound_ to any command, including `noop`, it is
no longer forwarded to other keymaps nor the renderer.

So `nothing` might be clearer here.

**Comment by pdelfino** [comment]
I am closing this issue since the suggestions have now been merged with  PR #1672.
